third-party/arcanum-worlds/odyssey-of-the-dragonlords/race-thyleansiren.xml

```diff
@@ -0,0 +1,121 @@
+    <rules>
+      <stat name="dexterity" value="1"/>
+      <stat name="charisma" value="2"/>
+      <stat name="innate speed" value="30" bonus="base"/>
+      <grant type="Size" id="ID_SIZE_MEDIUM" />
+      <grant type="Language" id="ID_LANGUAGE_COMMON" />
+      <grant type="Language" id="ID_LANGUAGE_CELESTIAL" />
+      <grant type="Racial Trait" id="ID_AW_OOTD_RACIAL_TRAIT_THYLEAN_SIREN_ENTHRALLING_VOICE" />
+      <grant type="Racial Trait" id="ID_AW_OOTD_RACIAL_TRAIT_THYLEAN_SIREN_WAVERING_EMOTIONS" />
+      <grant type="Racial Trait" id="ID_AW_OOTD_RACIAL_TRAIT_THYLEAN_SIREN_FLIGHT" />
+      <grant type="Racial Trait" id="ID_AW_OOTD_RACIAL_TRAIT_THYLEAN_SIREN_SONGS_OF_SORROW" />
+    </rules>
+  </element>
+
+  <element name="Enthralling Voice" type="Racial Trait" source="Odyssey of the Dragonlords" id="ID_AW_OOTD_RACIAL_TRAIT_THYLEAN_SIREN_ENTHRALLING_VOICE">
+    <description>
+      <p>You have advantage on Performance and Persuasion checks made with your voice. Additionally, your powerful lungs allow you to hold your breath for up to 1 hour.</p>
+    </description>
+    <sheet>
+      <description>You have advantage on Performance and Persuasion checks made with your voice. Additionally, your powerful lungs allow you to hold your breath for up to 1 hour.</description>
+    </sheet>
+  </element>
+
+  <element name="Wavering Emotions" type="Racial Trait" source="Odyssey of the Dragonlords" id="ID_AW_OOTD_RACIAL_TRAIT_THYLEAN_SIREN_WAVERING_EMOTIONS">
+    <description>
+      <p>Your mood affects your ability to sing and fly. After any short or long rest, you must choose whether you are feeling joyful or sad. While you are feeling sad, you lose your flying speed but gain <i>songs of sorrow</i>. While you are feeling joyful, you gain your flying speed but lose <i>songs of sorrow</i>. Your mood may change before the next time you rest, but it won’t affect which ability you have access to until after your next rest is completed.</p>
+    </description>
+    <sheet>
+      <description>Your mood affects your ability to sing and fly. After any short or long rest, you must choose whether you are feeling joyful or sad. While you are feeling sad, you lose your flying speed but gain <i>songs of sorrow</i>. While you are feeling joyful, you gain your flying speed but lose <i>songs of sorrow</i>. Your mood may change before the next time you rest, but it won’t affect which ability you have access to until after your next rest is completed.</description>
+    </sheet>
+  </element>
+
+  <element name="Flight" type="Racial Trait" source="Odyssey of the Dragonlords" id="ID_AW_OOTD_RACIAL_TRAIT_THYLEAN_SIREN_FLIGHT">
+    <description>
+      <p>You have a flying speed of 30 feet. To use this speed, you can’t be wearing medium or heavy armor.</p>
+    </description>
+    <sheet>
+      <description>You have a flying speed of 30 feet. To use this speed, you can’t be wearing medium or heavy armor.</description>
+    </sheet>
+  </element>
```

Comment:
innate fly speed statistic can be added (a statistic rule can have equipped requirements for wearing armor, see [aarakocra ](https://github.com/aurorabuilder/elements/blob/master/supplements/princes-of-the-apocalypse/races/race-aarakocra.xml#L77-L87)for example)